src/com/twitter/intellij/pants/settings/PantsProjectSettingsControl.java

```diff
@@ -77,6 +79,12 @@ protected void fillExtraControls(@NotNull PaintAwarePanel content, int indentLev
     myUseIdeaProjectJdkCheckBox.setSelected(mySettings.useIdeaProjectJdk);
     myImportSourceDepsAsJarsCheckBox.setSelected(mySettings.importSourceDepsAsJars);
     myUseIntellijCompilerCheckBox.setSelected(mySettings.useIntellijCompiler);
+    LinkLabel<?> intellijCompilerHelpMessage = LinkLabel.create(PantsBundle.message("pants.settings.text.use.intellij.compiler.help.messasge"), new Runnable() {
```

Comment:
I can't quite see this message in the UI, it doesn't show up. Where should I be looking?

Reply:
Updated the description under 1).